### PR TITLE
Include karton metrics in the readme

### PR DESCRIPTION
**Pull request by nazywam**
Ref: https://github.com/CERT-Polska/karton/issues/128

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +18,16 @@ $ karton-dashboard run -h 127.0.0.1 -p 5000
 
 The `karton-dashboard` is just a wrapper on the `flask` program, and it works with any arguments accepted by flask. For example `karton-dashboard --help`, or `karton-dashboard run -h 0.0.0.0 -p 1234`. See [flask documentation](https://flask.palletsprojects.com/en/1.1.x/cli/) for more information.
 
+## Metrics
+
+Karton keeps track of consumed, produced, crashed tasks for each queue.
```
Comment (chivay) [inline README.md:23]:
```suggestion
Karton tracks number of consumed, produced and crashed tasks for each service (identity).
```

**Review by chivay — APPROVED** [review]

**Comment by rakovskij-stanislav** [comment]
Wow, thank you! We probably need to add a count of replicas in metricas to have a vision about replica correlation and task queue.

**Comment by rakovskij-stanislav** [comment]
For now we have task-oriented data: count of assigned, consumed, crashed, garbage-collected and produced.

**Comment by rakovskij-stanislav** [comment]
And I ask you to add prometheus expressions used in screenshots for faster dashboard creation 'cause some people will meet pql the first time :P

**Pull request merged**